torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -403,10 +410,14 @@ void ScriptModuleSerializer::serialize(
 
 void ScriptModuleSerializer::writeArchive(
     const IValue& value,
-    const std::string& archive_name) {
+    const std::string& archive_name,
+    const std::string& archive_dir,
+    const std::string& tensor_dir,
+    bool tensor_cdata_naming_scheme) {
```

Comment:
I don't love that we are forking the tensor naming scheme here but I guess following the principle of "don't change existing serialization…yet" it is necessary. We definitely need to follow up to unify all this though.